tox.ini

```
@@ -48,6 +50,7 @@ commands =
 
 [testenv:lint]
 description = Enforce quality standards under {basepython} ({envpython})
+base_python = python3.9
```

```suggestion
base_python = python3.10
```
since 3.9 support has been removed